scenarios/01_Breaking_the_Circle.cfg

```diff
@@ -322,8 +351,17 @@
 #else
         type=Lieutenant
         recruit=Peasant,City Militia
-#endif
-        {GOLD 90 100 110}
+#ifdef NIGHTMARE
+        extra_recruit=Fencer,Thug
```

Comment:
hm... how about Heavy Infantrymen? The point is that I wanted to give them another human unit with only melee attacks, and didn't really think it would make sense for them to have horses... Or how about thieves?

Reply:
Okay Heavy Infantrymen works.

Reply:
OK I went with them in [086c1d2](https://github.com/nemaara/A_New_Order/pull/53/commits/086c1d25c13ea2fc5549ae746b68eb2476cc283a)